src/utils/hooks.js

```diff
@@ -19,3 +24,92 @@ export function useStateWithStorage(storageKey, initialValue) {
 
 	return [value, setValue];
 }
+
+export function useUrgency(items) {
+	const [urgencyObject, setUrgecyObject] = useState({
+		overdue: [],
+		soon: [],
+		kindOfSoon: [],
+		notSoon: [],
+		inactive: [],
+	});
+
+	const getUrgency = (name) => {
+		const statusArray = Object.entries(urgencyObject).find(
+			([status, items]) => {
+				if (items.some((item) => item.name === name)) {
+					return status;
+				}
+			},
+		);
```

Comment:
The find callback should return a boolean indicating whether the current element satisfies the condition, rather than returning status, which is a string. Returning a boolean aligns with the intended use of the find method!